### DISCUSSION TITLE
v3.0.0-rc.4

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#17370] [Announcements]
The fourth release candidate of Nuxt 3 is out :

https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.4

This release contains commutative enhancements from RC.3. Thanks to all of the contributors to the code and documentation :green_heart: 

Expect another exciting release this week with new features :sparkles:

**Comment by tiborpino** [comment]
Getting error when yarn build

`Identifier 'createError' has already been declared `

**Reply by misaon** [reply]
![image](https://user-images.githubusercontent.com/15806356/173522870-191cccf3-aba0-4615-acc2-c72315303f57.png)
same problem during yarn build

**Reply by misaon** [reply]
https://github.com/nuxt/framework/issues/5454

**Comment by zavvla** [comment]
Hi, I see that the release dates have changed? Previously promised in June, now in summer?

**Reply by pi0** [reply]
It is what @Atinux  also announced at Vue.js Ammsterdam. Hopefully, it happens by July BTW.

**Reply by amiria703** [reply]
Hi @pi0 ! What is _Vue.js Ammsterdam_?

**Reply by danielroe** [reply]
[This conference](https://vuejs.amsterdam/).

**Comment by rodrigogs** [comment]
And SPA still not working:
https://github.com/nuxt/bridge/issues/27

**Comment by tkjaergaard** [comment]
We're experiencing the same issue when building: `Identifier 'createError' has already been declared`

**Comment by StanleyMasinde** [comment]
Hi, I'm facing this issue after upgrading the this release when running the dev server.

I'm using vite. Anyone else facing this? I don't think it should be happening that's why I did not create an issue.

### Edit
Not related to this release

<img width="938" alt="image" src="https://user-images.githubusercontent.com/35426405/173776325-c3deae0f-78e1-4099-aa26-a2a3bfc4b380.png">

**Reply by danielroe** [reply]
Best thing would be to create a new issue with a reproduction so we can look into it.

**Reply by StanleyMasinde** [reply]
Thank you for your response @danielroe. 

I figured it out the problem is not related to this release I had accidentally included ```<script lang="js">```

<img width="294" alt="image" src="https://user-images.githubusercontent.com/35426405/173780541-df800c07-bfd2-4f37-af32-c1e76477a668.png">

**Comment by zavvla** [comment]
please tell me I see a warning, but I don't understand what causes it? Is this how it should be?  In which direction to dig? thx

`<Suspense> is an experimental feature and its API will likely change.`

**Reply by danielroe** [reply]
This is coming from Vue 3 itself; it's not an issue with Nuxt or your project.

**Reply by StanleyMasinde** [reply]
It's just was warning from Vuejs core. It suspense was introduced with <script setup> 

You can read more [here](https://vuejs.org/guide/built-ins/suspense.html).

You don't have to do anything

**Comment by zihaoyy** [comment]
The official website said that the official version of midsummer will be released, and June is about to pass. Will it be released in these two days?

**Reply by eMeRiKa13** [reply]
June is just the beginning of summer. Midsummer is in august.

**Reply by zihaoyy** [reply]
> 

Well, maybe I misunderstood it, I understood it as solar terms
